yarn.lock

```diff
@@ -850,9 +850,9 @@ atob@^2.1.2:
   resolved "https://registry.yarnpkg.com/atob/-/atob-2.1.2.tgz#6d9517eb9e030d2436666651e86bd9f6f13533c9"
   integrity sha512-Wm6ukoaOGJi/73p/cl2GvLjTI5JM1k/O14isD73YML8StrH/7/lRFgmg8nICZgD3bZZvjwCGxtMOD3wWNAu8cg==
 
-auth0-js@^9.19.2:
+auth0-js@^9.20.0:
   version "9.20.0"
-  resolved "https://registry.yarnpkg.com/auth0-js/-/auth0-js-9.20.0.tgz#80d140a6741f677f5295f2b38e750e23961726fe"
+  resolved "https://a0us.jfrog.io/a0us/api/npm/npm/auth0-js/-/auth0-js-9.20.0.tgz#80d140a6741f677f5295f2b38e750e23961726fe"
```

Comment:
We can't use dependencies from Artifactory as no-one outside of the organization will be able to install it, it's also causing the Circle build to fail.

Reply:
Of course. 🤦‍♀️